### PR TITLE
 Product Approval Process + Merchant Context

### PR DESCRIPTION
**Pull request by gibmirmilch**
## PR Description

https://spryker.atlassian.net/browse/MP-3934

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+## Approving and denying the marketplace products
+
+Depending on the current status of the product, you can approve, deny or get the product in the *draft* status.
+To update the approval status of the product, in the _Actions_ column of the abstract product do the following:
+
+- For the product with status *Draft*:
+  - click **Send for Approval** to initiate the approval process.
+  - click **Approve** to approve the product.
```
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:73]: when it's a draft, there is no action buttons, it is waiting for the merchant to send for approval

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  - click **Deny** to reject the product.
```
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:74]: when it's a draft, there is no action buttons, it is waiting for the merchant to send for approval

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+title: Marketplace Product Approval Process feature overview
+description: This document contains concept information for the Marketplace Product Approval Process feature.
+template: concept-topic-template
+---
+The [marketplace operator](/docs/marketplace/user/intro-to-the-spryker-marketplace/back-office-for-marketplace-operator.html) is primarily responsible for ensuring the quality of data on the marketplace, including merchants, products, and offers. To control those things in the Marketplace, the approval mechanism is a key feature.
+
+The Marketplace Product Approval Process allows marketplace owners to verify the products of merchants before showing them to customers.
+
+Thus, only the approved and active products within the validity dates range and existing for the defined store are displayed in the Storefront.
```
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:10]: Thus, only the approved, active, within the validity dates range, and existing for the defined store are displayed in the Storefront.

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+{% endinfo_block %}
+
+A product can have one of the following statuses:
+
+| STATUS               | DESCRIPTION                                                  |
+| -------------------- | ------------------------------------------------------------ |
+| Draft                | When the product is created, it obtains the *draft* status.  |
+| Waiting for approval | When the product is sent for approval by a merchant in the Merchant Portal, the status changes to *waiting for approval*. |
+| Approved             | The marketplace administrator can  approve the product in the Back Office and in this case the status of the product will change to the *approved* one in the Merchant Portal. The marketplace administrator can cancel the approval procedure and and return the product into *draft* status. |
```
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:26]: and also deny

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+| Denied               | When the marketplace administrator rejects the product in the Back Office, the product gets *denied* status in the Merchant Portal. The marketplace administrator can cancel the approval procedure and and return the product into *draft* status. |
```
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:27]: and approve

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+{% info_block infoBox "Info" %}
+
+You can configure the logic of the statuses on the project level.
+
+{% endinfo_block %}
+
+A product is only displayed in the Storefront if it is *approved* and its status is *Active*.
```
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:35]: approved, active, within the validity dates range, and existing for the defined store is displayed in the Storefront.

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+## Marketplace Product Approval process workflow
+
+In order for the marketplace product to be displayed in the Storefront, every [merchant](/docs/marketplace/user/features/{{page.version}}/marketplace-merchant-feature-overview/marketplace-merchant-feature-overview.html) needs this product to be approved by the marketplace administrator. The approval process goes through the following stages:
+
+1. **Submitting the product for approval.** The [merchant user](https://docs.spryker.com/docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-personas.html) submits the request for product approval in the Merchant Portal. The status of the product changes to *Waiting for approval*.
```
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:41]: first creating a product and staus is draft

File: docs/marketplace/user/merchant-portal-user-guides/202200.0/products/abstract-products/managing-marketplace-abstract-product.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+| Attributes  | In this pane, you can manage attributes for your product. See [Managing abstract product attributes](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/managing-marketplace-abstract-product-attributes.html) for more details. |           |
+| Categories  | Defines the [categories](/docs/scos/user/features/{{page.version}}/category-management-feature-overview.html) the product is displayed in. |           |
+| SEO         | In this pane, you can manage meta information for your product. See [Managing abstract product meta information](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/managing-marketplace-abstract-product-meta-information.html) for more details. |           |
+
+### Sending the product for approval
+
+In order for the product to be available on the Storefront, it needs to be approved. To send the product for approval:
+
+1. Next to the abstract product, you want to edit, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *[Product name]* drawer, *Abstract Product Details* tab.
```
Comment (AdvaBD) [inline docs/marketplace/user/merchant-portal-user-guides/202200.0/products/abstract-products/managing-marketplace-abstract-product.md:68]: we will not have for already existing products, only new, and this doc is missing approval process when creating a new product by merchants in the merchant portal

**Review by AdvaBD — CHANGES_REQUESTED** [review]

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+See the _References_ section to see the examples of how the attributes look like.
+
+
+## Approving and denying the marketplace products
+
+Depending on the current status of the product, you can approve, deny or get the product in the *draft* status.
+To update the approval status of the product, in the _Actions_ column of the abstract product do the following:
+
+- For the product with status *Draft*:
+  - click **Send for Approval** to initiate the approval process.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:72]: Currently, we do have this option in the back office: https://prnt.sc/sEZciJ_gqwqG

Until we have a separate instance of B2B demo shop, I'll leave it as it is, and will change before the release.

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md (hunk 1 above)
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:73]: prevoius comment https://prnt.sc/sEZciJ_gqwqG

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md (hunk 2 above)
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:74]: prevoius comment https://prnt.sc/sEZciJ_gqwqG

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md [hunk 10]
```diff
@@ -1,64 +1,91 @@
...
+
+## Approving and denying the new marketplace products
+
+Once the product is created, it needs to be approved by the marketplace administrator.
+
+Depending on the current status of the product, you can approve, deny or get the product in the *draft* status.
+To update the approval status of the product, in the _Actions_ column of the abstract product do the following:
+
+- For the product with status *Draft*:
+  - click **Send for Approval** to initiate the approval process.
```
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:73]: doesn't exist in demo shops

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md [hunk 11]
```diff
@@ -1,64 +1,91 @@
...
+  - click **Approve** to approve the product.
+  - click **Deny** to reject the product.
+
+- For the product with status *Waiting for Approval*:
+  - click **Back to Draft** to revert the product to *Draft* status.
+  - click **Approve** to approve the product.
+  - click **Deny** to reject the product.
+
+- For the product with status *Approved*:
+  - click **Back to Draft** to revert the product to *Draft* status.
```
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:83]: doesn't exist in demo shops

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md [hunk 12]
```diff
@@ -1,64 +1,91 @@
...
+  - click **Deny** to reject the product.
+
+- For the product with status *Denied*:
+  - click **Back to Draft** to revert the product to *Draft* status.
```
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:87]: doesn't exist in demo shops

**Review by AdvaBD — CHANGES_REQUESTED** [review]

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md [hunk 13]
```diff
@@ -1,64 +1,91 @@
...
+    Repeat the procedure if needed.
+5. Click **Save**.
+
+See the _References_ section to see the examples of how the attributes look like.
+
+## Approving and denying the new marketplace products
+
+Once the product is created, it needs to be approved by the marketplace administrator.
+
+Depending on the current status of the product, you can approve, deny or get the product in the *draft* status.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:69]: removed this part

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md (hunk 10 above)
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:73]: removed it from the instructions

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md [hunk 14]
```diff
@@ -1,64 +1,91 @@
...
+To update the approval status of the product, in the _Actions_ column of the abstract product do the following:
+
+- For the product with status *Draft*:
+  - click **Send for Approval** to initiate the approval process.
+  - click **Approve** to approve the product.
+  - click **Deny** to reject the product.
+
+- For the product with status *Waiting for Approval*:
+  - click **Back to Draft** to revert the product to *Draft* status.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:78]: removed

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md (hunk 11 above)
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:83]: removed

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md [hunk 15]
```diff
@@ -1,81 +1,89 @@
...
-This document describes the actions a merchant can do in the **Offers** section of the Merchant Portal.
+This topic describes the actions a Merchant can do in the Offers section in the Merchant Portal.
 
 ## Prerequisites
 
-To start managing product offers, navigate to the **Merchant Portal&nbsp;<span aria-label="and then">></span> Offers**.
+To start managing product offers, navigate to the **Merchant Portal** > **Offers**.
 
-Review the reference information in this article before you start, or look up the necessary information as you go through the process.
+Review the reference information in this article before you start, or just look up the necessary information as you go through the process.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md:13]: please remove "just": https://developers.google.com/style/word-list#letter-j

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md [hunk 16]
```diff
@@ -1,81 +1,89 @@
...
 
 ## Creating a product offer
 
-To create a product offer, take the following steps:
+To create a product offer:
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md:17]: we should introduce procedures with the full sentence: https://developers.google.com/style/procedures

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md [hunk 17]
```diff
@@ -1,81 +1,89 @@
...
+
 1. Click **Add Offer**.
-2. From the list of concrete products, select the product you want to create an offer for.
-  The **Create Offer** drawer opens.
+2. From the list of concrete products, select the  product you want to create an offer for.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md:20]:
```suggestion
2. From the list of concrete products, select the product you want to create an offer for.
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md [hunk 18]
```diff
@@ -1,81 +1,89 @@
...
 3. Select **Save**.
 
 ![img](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Merchant+Portal+user+guides/Offers/edit-offers.gif)
 
+
+
 ## Reference information: Creating and editing product offers
 
-This section describes attributes you see when creating and editing product offers.
+This section describes attributes you see when creating and editing product offer.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md:59]:
```suggestion
This section describes attributes you see when creating and editing a product offer.
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md [hunk 19]
```diff
@@ -9,54 +9,77 @@ This document describes how to manage marketplace abstract products in the Merch
 
 ## Prerequisites
 
-To start working with marketplace abstract products, go to the **Merchant Portal&nbsp;<span aria-label="and then">></span> Products**.
+To start working with marketplace abstract products, go to the **Merchant Portal** > **Products**.
 
-This document contains reference information. Make sure to review it before you start, or look up the necessary information as you go through the process.
+This article contains reference information. Make sure to review it before you start, or just look up the necessary information as you go through the process.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md:14]: Please consider revering to "document" and without "just"

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md [hunk 20]
```diff
@@ -9,54 +9,77 @@ This document describes how to manage marketplace abstract products in the Merch
...
 To edit the existing marketplace abstract product:
 
-1. Next to the abstract product, you want to edit, hover over the three dots and click **Manage Product** or just click the line. This takes you to the **_[Product name]_**, **Abstract Product Details** tab.
-2. In the **Name & Description** pane, edit **Name** and **Description** for every locale.
-3. In the **Stores** pane, in the drop-down menu select the stores where the product is available.
-4. In the **Tax Set** pane, in the drop-down menu, select the necessary tax set.
-5. In the **Categories** pane, in the drop-down menu, select the categories where your product belongs.
+1. Next to the abstract product, you want to edit, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *[Product name]*, *Abstract Product Details* tab.
+2. In the *Name & Description* pane, edit *Name* and *Description* for every locale.
+3. In the *Stores* pane, in the drop-down menu select the stores where the product is available.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md:44]:
```suggestion
3. In the *Stores* pane, in the drop-down menu, select the stores where the product is available.
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md [hunk 21]
```diff
@@ -9,54 +9,77 @@ This document describes how to manage marketplace abstract products in the Merch
...
+
+## Sending the product for approval
+
+{% info_block warningBox "Warning" %}
+
+You can approve only the [newly created marketplace product](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/creating-marketplace-abstract-product.html).
+
+{% endinfo_block %}
+
+In order for the new product to be available on the Storefront, it needs to be approved. To send the product for approval:
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md:72]:
```suggestion
For the new product to be available on the Storefront, it needs to be approved. To send the product for approval, do the following:
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 22]
```diff
@@ -9,20 +9,19 @@ This document describes how to edit a marketplace concrete product in the Mercha
...
 ## Prerequisites
 
-To start working with marketplace concrete products, take the following steps:
-1. Go to the **Merchant Portal&nbsp;<span aria-label="and then">></span> Products**.
-2. Hover over the three dots next to the abstract product for which you manage a concrete product and click **Manage Product** or just click the line. The **[Product name]** drawer opens.
-3. Navigate to the **Concrete Products** tab.
+To start working with marketplace concrete products, go to the **Merchant Portal** > **Products**.  Hover over the three dots next to the abstract product for which you manage a concrete product and click **Manage Product** or just click the line. This takes you to the *[Product name]* drawer. Navigate to the *Concrete Products* tab.
 
-This document contains reference information. Make sure to review it before you start, or look up the necessary information as you go through the process.
+This article contains reference information. Make sure to review it before you start, or just look up the necessary information as you go through the process.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md:14]: same as above

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 23]
```diff
@@ -9,20 +9,19 @@ This document describes how to edit a marketplace concrete product in the Mercha
...
 
 ## Filtering and sorting concrete products
 
-The following drop-down menus can be used to filter the marketplace concrete products on the **Concrete Products** page:
+The following drop-down menus can be used to filter the marketplace concrete products on the *Concrete Products* page:
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md:18]:
```suggestion
You can use the following drop-down menus to filter the marketplace concrete products on the *Concrete Products* page:
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md [hunk 24]
```diff
@@ -34,42 +33,43 @@ By default, the table is sorted descendingly by SKU.
...
 To activate a marketplace concrete product:
-1. On the **Concrete Products** page, next to the concrete product you want to activate, hover over the three dots and click **Manage Product** or just click the line. The **Concrete Product SKU, Name** page opens.
-2. In the **Status** pane, select **Concrete Product is online** to make the concrete product online.
+
+1. On the *Concrete Products* page, next to the concrete product you want to activate, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *Concrete Product SKU, Name* page.
+2. In the *Status* pane, select *Concrete Product is online* to make the concrete product online.
 3. Click **Save**.
 
-To deactivate the product, clear **Concrete Product is online** to make the concrete product offline.
+To deactivate the product, deselect *Concrete Product is online* to make the concrete product offline.
```
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md:41]: Please revert to "clear": https://developers.google.com/style/word-list#letter-d

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/marketplace/user/features/202204.0/marketplace-product-approval-process-feature-overview.md [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+
+The *Marketplace Product Approval Process* allows marketplace owners to verify the products of merchants before showing them to customers.
+
+Thus, the Storefront displays only the approved, active, within the validity period, and existing for the defined store products.
+
+## Product lifecycle
+
+{% info_block warningBox "Note" %}
+
+The following workflow is valid only in case the Marketplace Product Approval Process feature is enabled.
```
[the remark this answers is not in the document]
Comment (gibmirmilch) [reply docs/marketplace/user/features/202204.0/marketplace-product-approval-process-feature-overview.md:16]: added

**Review by lenadoc — APPROVED** [review]

**Pull request merged**